test/spec.js

```diff
@@ -725,4 +726,69 @@ describe('checkDependencies', function () {
             });
         }
     });
+
+    describe('cli', function () {
```

Comment:
I'd like to have at least 2 test (success & failure) that use regular spawned process runnin `./bin/cli.js` with proper parameters to be sure that not only the "CLI API" works but the command itself.